src/app/admin/feedback/page.tsx

```
@@ -0,0 +1,171 @@
+									</DropdownMenuContent>
+								</DropdownMenu>
+							</div>
+						</div>
+
+						<div className="bg-muted/30 p-4 rounded-xl border border-dashed border-muted-foreground/20 italic text-sm relative">
+							<MessageSquare className="size-4 text-muted-foreground/30 absolute -top-2 -left-2 bg-background rounded-full" />
+							{item.text}
+						</div>
+
+						<div className="flex items-center justify-between mt-auto pt-4 border-t text-[10px] text-muted-foreground">
+							<div className="flex items-center gap-1.5 hover:text-primary transition-colors cursor-pointer" onClick={() => window.open(item.pageUrl, '_blank')}>
```

Biome (2.1.2)

[error] Static Elements should not be interactive.

To add interactivity such as a mouse or key event listener to a static element, give the element an appropriate role value.

(lint/a11y/noStaticElementInteractions)